.github/workflows/markdown-lint.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: markdown-lint
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
I'd remove this since we're not using it.

Reply:
done